docs/DeveloperGuide.md

```diff
@@ -632,3 +632,12 @@ testers are expected to do more *exploratory* testing.
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+
+## **Appendix: Planned Enhancements**
+
+Team size: 5
+
+1. **Switch to the tab of data modified**
+
+Currently, the tab on the bottom of the details panel always switches to the 'Weight' tab when any details of a person is updated. We plan to make the tab switch to the 'Weight' tab when weight is modified, and the 'Exercises' tab when exercises are modified. Otherwise, the tab should not change when any other fields are modified.
```

Comment:
```suggestion
Currently, the tab on the bottom of the details panel always switches to the 'Weight' tab when any details of a client is updated. We plan to make the tab switch to the 'Weight' tab when weight is modified, and the 'Exercises' tab when exercises are modified. Otherwise, the tab should not change when any other fields are modified.
```